### DISCUSSION TITLE
lolMiner 1.64

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1802] [General]
### Changes

- Improved Kaspa only mining performance. Speed increase is about 8-8.5% on Nvidia Pascal GPUs, 4.5-5% on Nvidia Turing and Ampere GPUs and 3-4% on AMD Navi and Big Navi GPUs
- Beta feature: added options to set core clock offset (**--coff**), memory clock offset (**--moff**), power limit (**--pl**) and a fixed fan speed (**--fan**) on common Nvidia GPUs. Required are admin privileges and Nvidia **drivers 520 or higher**!
The syntax is the same as with --cclk and --mclk - if a single value is given then it will be applied to all compatible GPUs, else a coma separated list of values can be given using a \* character to skip over GPUs.
- Added a new parameter --no-oc-reset to turn off the reset of overclock settings when ending the miner.
- Windows: Added a beta gui to generate overclock settings strings / .bat files for the miner. Also the tool can apply the chosen settings directly.

### Fixes
- Fixing a bug with ETHV1 (nicehash) stratum mode that may cause the worker name to be appended twice when it was given by --user <wallet>.<workerName> (the use of --worker did not have this issue).  

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.64'>lolMiner 1.64</a>.</em>

**Comment by VDV777** [comment]
settins on linux: 
rtx 3090 - kaspa - 1410 CClock 8100 MClock 130w PL = 850mh 130w
settins on windows from new GUI App: 
rtx 3090 - kaspa - 1410 CClock 8100 MClock 130w PL = 230mh 130w = LOLLLLL MINER((((
p.s. please fix it

**Reply by Cybertiron** [reply]
I am minning 1500 CCLK, 810 MCLK, Coff 300-400 works like a charm. I tried 1600 CCLK but efficiency lowers. 1400 Is bit worse efficiency for me. Why you use 8100 that is too much for core intensive ? and you didnt add coff at least 300

**Reply by VDV777** [reply]
5000 or 8100 nothing matter

**Reply by VDV777** [reply]
rly work! thanks dude! I thought i can't set memory clock value less than 5000mhz.
p.s. i apologize from lolminer developers!!!

**Reply by avepdgk** [reply]
how many do you have Mh/s in windows? On CCLK 1470 COFF 300 MCLK 810 PL 130 i have only ~600 on my 3090

**Reply by VDV777** [reply]
i used PL 140w limit and 1500 CCLK COFF 300 MCLK 810: i got : 840 - 900 mh. 900mh on aorus and evga and 3080TI IChill, etc. 840-850 mh

**Comment by FVG01** [comment]
como configuro esse parâmetros --cclk e --mclk no minerstat

**Comment by ghost** [comment]
on windows, where to store this code "--no-oc-reset", this code for reset oc after end miner right? because after i try the oc on .bat and stop miner i see on afterburner the oc was lock because i have lock the oc on the .bat

**Comment by Lolliedieb** [comment]
The switch --no-oc-reset is meant to turn off the reset when endimg miner.

Note that the reset currently only takes place when ending the miner in a
clean way i.e. ending it with ctrl+c. If you just hit the x on the terminal
window it currently can happen the miner exits more abruptly and fails to
reset the oc.

akselgp19 ***@***.***> schrieb am Mi., 7. Dez. 2022, 03:36:

**Reply by Cybertiron** [reply]
Does it really work? I tried to add to batch file and turn it off cleanly, but nothing is changed.

**Reply by Cybertiron** [reply]
I just created a different bat: CCLK 0, MCLK 0, COFF 0
Seems to reset everything. It is not so clean, but does the job instead of restarting rig
